hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServer.java

```diff
@@ -53,6 +54,8 @@
   private Timer bulkLoadTimer;
   private Meter serverReadQueryMeter;
   private Meter serverWriteQueryMeter;
+  protected long slowMetricTime = 1000L;
+  protected static final int DEFAULT_SLOW_METRIC_TIME = 1000; // milliseconds
```

Comment:
Where do we read slowMetricTime from the configuration? I don't see it in this PR

Reply:
sorry, the pr has updated!  please review again!

Reply:
@saintstack  hi, pr updated already!